trunk/SUAVE/Methods/Propulsion/propeller_design.py

```diff
@@ -11,13 +11,14 @@
 
 import SUAVE
 import numpy as np
-from SUAVE.Core import Units
-
+from SUAVE.Core import Units , Data
+from SUAVE.Methods.Geometry.Two_Dimensional.Cross_Section.Airfoil.import_airfoil_geometry \
+     import import_airfoil_geometry
 # ----------------------------------------------------------------------
 #  Propeller Design
 # ----------------------------------------------------------------------
-    
-def propeller_design(prop_attributes):
+
+def propeller_design(prop,N=20):
```

Comment:
Write out what N is

Reply:
N is the number of stations of the propeller. I will spell it out